app/controllers/collections_controller.rb

```diff
@@ -55,6 +55,8 @@ def create
       collection_metadata = params[:collection_metadata]
       collection_name = params[:name]
       licence_id = params[:licence_id].present? ? params[:licence_id] : nil
+      privacy = true
+      privacy = (params[:private].to_s.downcase == 'true') unless params[:private].nil?
       if collection_name.blank? || collection_name.length > 255 || collection_metadata.nil?
```

Comment:
I've addressed this in a separate pull request (since this one was already merged) https://github.com/IntersectAustralia/hcsvlab/pull/52